src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -546,7 +546,7 @@ private ITypeSymbol GetAwaitInfo(SemanticModel semanticModel, ISyntaxFactsServic
         {
             var awaitExpression = FirstAwaitExpressionAncestor(syntaxFactsService, node);
 
-            var innerExpression = syntaxFactsService.GetExpressionOfAwaitExpression(node);
+            var innerExpression = syntaxFactsService.GetExpressionOfAwaitExpression(awaitExpression);
```

Comment:
@heejaechang @CyrusNajmabadi @jinujoseph This was actually flagged by the new unused value analyzer: https://github.com/dotnet/roslyn/pull/31387/files#diff-ae8841390c6083969313ad6d1cf9bad4R546, my bad not to follow up on it. I will try to run the analyzer on IDE layers to clean up these issues and/or file tracking bugs if the desired behavior is unclear instead of a trivial typo.

Reply:
Sounds good.